README.md

```diff
@@ -1,6 +1,306 @@
+These are detected by the deployment script and required for setup.
+
+
+`# gcloud config set account <user account>`
+
+`# gcloud config set project <project>`
+
+`# gcloud config set compute/zone <zone>`
+
+
+### Step 1: Deploy Gluster-Kubernetes cluster in Google Compute Engine (GCE)
+This step sets up the **External Service Provider** portion of our topology (see [diagram](docs/diagram/control-diag.md)).
+
+To kick off deployment, run `gk-cluster-deploy/deploy/cluster/gk-up.sh`
+The script has a number of configurable variables relative to GCE Instance settings
+They can be found in `deploy/cluster/lib/config.sh`  These can be overridden inline with `gk-up.sh` or as environment variables. To run the script with out pausing run`gk-up.sh -y`.
+
+Runtime takes around 5 to 10 minutes
+Go get some coffee.
+
+Alternatively, you can manually deploy the GCE portion by following [these instuctions](//TODO) // TODO
+
+When deployment completes, commands to ssh into the master node and the URL of the CNS Object Broker will be output. **Note the URL and PORT.**
+
+### Step 2: Deploy the Service-Catalog
+This step sets up the **K8s Cluster** portion of our topology (see [diagram](docs/diagram/control-diag.md)).
+
+*In a separate terminal:*
+1. Change directories to the `service-catalog` repository.
+
+2. Follow the [Service-Catalog Installation instructions](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/introduction.md#installation)
+Once the Service-Catalog is deployed, return here.
+
+
+## Using the Service Catalog
+
+**STOP!** If you have made it this far, it is assumed that you now have
+
+- A 4 node Kube cluster in GCE, running:
+  - Gluster-Kubernetes and its S3 components.
+  - Our CNS Object Broker.
+
+You can check the status of all these components at once by executing
+
+`# gcloud compute ssh <master node name> --command="kubectl get pod,svc --all-namespaces"`
+
+### Create the *ServiceBroker* (API Object)
+
+Change to the `cns-object-broker` directory created when cloning the repo.
+
+0. Retrieve the URL and PORT of the cns-object-broker.
+
+    If `gk-up.sh` was run, it will have been output at the end of the script
+
+
+    To get the url and port manually, first note the **external ip** of any GCE node in the cluster
+    The broker is exposed via a *NodePort Service* and so is reachable via any node.
+
+    `# gcloud compute instances list --filter="<user name>"`
+
+    Next, get the port exposed by the *NodePort Service*.
+
+    `gcloud compute ssh <master node name> --command="kubectl get svc -n broker broker-cns-object-broker-node-port"`
+
+    ```
+    NAME                                 CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    broker-cns-object-broker-node-port   10.102.63.165   <nodes>       8080:32283/TCP   1d
+    ```
+
+    The ports are formatted as \<InternalPort\>:\<ExternalPort\>
+    Note the ExternalPort.
+
+1.  Edit *examples/service-catalog/service-broker.yaml*
```

Comment:
"Back on the local k8s cluster, edit *examples/service-catalog/service-broker.yaml*"

Reply:
ditto above comment